src/routes_initializer.coffee

```diff
@@ -17,12 +17,14 @@ class RoutesInitializer
 
   constructor: (@routesFile, @reverseRoutes, @addRoute) ->
     @_routes = []
+    @_scope = ''
     require(@routesFile) @getRouteDirectives()
 
 
   # Returns a route directive for a specific http method to be called in a routes file
   getHttpDirective: (httpMethod) ->
     (url, {to, as}) =>
+      url = "#{@_scope}#{url}"
```

Comment:
Oh yeah, good point. One nice thing about using path though is if the scope ends in a trailing slash it will fix that for us too